highlighters/main/main-highlighter.zsh

```diff
@@ -195,7 +195,13 @@ _zsh_highlight_main__stack_pop() {
 _zsh_highlight_highlighter_main_paint()
 {
   # Before we even 'emulate -L', save the user's options
-  local -A useroptions; useroptions=("${(@kv)options}")
+  local -A useroptions
+  if zmodload -e zsh/parameter; then
```

Comment:
Maybe `|| zmodload zsh/parameter`?

Reply:
`if zmodload -e zsh/parameter; then` is already used in the main highlighter. The driver attempts to load the module, so I don't see a reason to attempt again.